packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -144,7 +146,11 @@ const MyAdsRowRenderer = observer(({ row: advert }) => {
                             )}
                         </div>
                         <div className='p2p-my-ads__table-row-details'>
-                            <Text color='profit-success' line_height='m' size='xxs'>
+                            <Text
+                                color={is_advert_listed ? 'profit-success' : 'less-prominent'}
+                                line_height='m'
```

Comment:
nit: you can remove this line cause the default for line_height is m 👍 same with the others
```suggestion
```

Reply:
yes. there are other issues as well i found with this file. will do it as part of refactoring.